android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/model/HealthModule.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.appfeature.model
+
+enum class HealthModule {
```

Comment:
We are using `enum` to avoid string constants here, the configuration will be read from app feature configurations.

Reply:
This will later be renamed to `RegisterType`/`RegisterFactor` or something relevant as we had discussed when scoping mwcore register - the configuration is used to differentiate the various registers supported in the app.

Reply:
The specific modules are not something we want to define in the code but through configurations, we don't want the code to limit the potential modules